test/studies/ssca2/main/SSCA2_Modules/SSCA2_kernels.chpl

```diff
@@ -476,7 +476,8 @@ module SSCA2_kernels
 
 	const n_edges          = G.num_edges;
 	const N_VERTICES       = vertex_domain.size;
-	const N_START_VERTICES = if starting_vertices == G.vertices
+	const N_START_VERTICES = if chpl_sameDomainKind(starting_vertices, G.vertices) &&
```

Comment:
I wish I knew why the code author is comparing these two domains. Was it about comparing the index set, the domain kind, or just whether it was called from callsite 1 or callsite 2?

In order to get correct output, it may be that either of the `===` identity or .type comparison will do.